public/js/index.js

```diff
@@ -1,57 +0,0 @@
-(()=>{
-  dom();
-})();
```

Comment:
SO @ronanyeah what is this Function exactly and what is for:  
```javaScript
(function() {
  // dom code
})();
```
execute on load or What!!